test/e2e/app-dir/client-module-with-package-type/app/import-cjs/page.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
+
+import EsmFromCjs from 'lib-cjs'
```

Comment:
this is the case that matches `jotai`s scenario. other permutations are just for completeness